Sources/SWBCore/BuildRequestContext.swift

```diff
@@ -149,7 +149,7 @@ extension BuildRequestContext {
 
         let currentPlatformFilter = PlatformFilter(settings.globalScope)
 
-        // FIXME: It is a bit unfortunate that we need to compute all this for the `uniquingSuffix` behaviour.
```

Comment:
`Behaviour` is the correct spelling in British English. We may need to introduce a simple guideline to standardize either British or US spelling.

Reply:
I see you already mentioned it in the PR description. My point is to establish a clear code of conduct that explicitly requires running a tool like this across the code for any change, or simply making it mandatory for every change moving forward.

Reply:
I don't, no.